apps/wallet/tailwind.config.ts

```diff
@@ -31,6 +31,8 @@ export default {
                 15: '3.75rem',
                 'popup-height': '600px',
                 'popup-width': '360px',
+                'stake-card-height': '140px',
```

Comment:
Don't love hard-coding the width and height, there's almost always a better way

Reply:
I removed the hardcoded height and width